fabischolasi/fast-food-fast-v1
# #160799554 contains all user api endpoints for login and signup

**What does this PR do?**
   - This pr contains all the endpoints for the users manipulation

**How should this be manually tested?**
  - clone this branch on your local machine, and make sure you have postman running
  - add some funtionality so as you can run the app manually on bash
  - navigate to postman and put the right url specified then test to see if the endpoints are working
  - make sure you have postgresql installed

**Any background context you want to provide?**
  - use of flask_restful to build the endpoints

**What are the relevant pivotal tracker stories?**
  - #160799554
